frontend/src/components/WordBank.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+let animal = ["cat", "dog", "eagle", "buffalo", "stingray", "racoon", "giraffe", "cheetah", "pony", "squirrel", "eel", "frog", "chicken", "wolf", "crocodile", "owl", "platypus", "hamster", "snake", "pig", "zebra", "whale", "cow", "bat", "porcupine", "ant"];
```

Comment:
We want to use `const` instead of `let` here because we're not planning on inserting/removing any values from these lists, so `const` will ensure this. Please replace all instances of `let` with `const` in this file.